mapie/conformity_scores/conformity_scores.py

```diff
@@ -158,20 +184,148 @@ def get_conformity_scores(
 
         Parameters
         ----------
+        X: NDArray
+            Observed feature values.
+
         y: NDArray
-            Observed values.
+            Observed target values.
 
         y_pred: NDArray
-            Predicted values.
+            Predicted target values.
 
         Returns
         -------
         NDArray
             Conformity scores.
         """
-        conformity_scores = self.get_signed_conformity_scores(y, y_pred)
+        conformity_scores = self.get_signed_conformity_scores(X, y, y_pred)
         if self.consistency_check:
-            self.check_consistency(y, y_pred, conformity_scores)
+            self.check_consistency(X, y, y_pred, conformity_scores)
         if self.sym:
             conformity_scores = np.abs(conformity_scores)
         return conformity_scores
+
+    @staticmethod
+    def get_quantile(
+        values: NDArray,
+        alpha_np: NDArray,
+        axis: int,
+        method: str
+    ) -> NDArray:
+        """
+        Compute the alpha quantile of the conformity scores considering
+        the symmetrical property if so.
```

Comment:
UPD: the symmetrical property is not considered now.

Reply:
Done